readme.md

```diff
@@ -61,6 +61,17 @@ object.foo.bar = {x: 'y', y: 'x'};
 deleteProperty(object, 'foo.bar.x');
 console.log(object);
 //=> {foo: {bar: {y: 'x'}}}
+
+const object = {
+	foo: {
+		bar: ['👸🏻 You found me Mario!'],
+	},
+	'foo.bar[0]' : '🍄 The princess is in another castle!',
```

Comment:
I think the example should be realistic. It's unlikely anyone would use `[0]` in an actual key. Make it just `'foo.bar'`.